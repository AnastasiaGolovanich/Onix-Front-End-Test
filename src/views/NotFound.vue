<template lang="pug">
section(class="news")
  div(class="current-page")
    h1(class="not-found") Not Found
    p(class="not-found") Oops we couldn't find that page. Try going
      |
      |
      router-link(:to="{name: 'Tasks'}") home
</template>

<script lang="ts">
import { defineComponent } from 'vue'

export default defineComponent({
  name: 'NotFound'
})
</script>
<style scoped>
.not-found {
  opacity: 0.5;
  font-family: 'Open Sans', sans-serif;
  font-size: 20px;
  line-height: 60px;
  color: #131313;
  text-align: center;
}
</style>
