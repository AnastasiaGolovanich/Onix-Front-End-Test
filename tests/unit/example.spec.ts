import { shallowMount } from '@vue/test-utils'
import NotFound from "@/views/NotFound.vue";

describe('HelloWorld.vue', () => {
  it('renders props.msg when passed', () => {
    const msg = 'new message'
    const wrapper = shallowMount(NotFound, {
      props: { msg }
    })
    // expect(wrapper.vm.$data.isModalVisible).toBe(true)
    expect(wrapper.element).toMatchSnapshot();
  })
})
