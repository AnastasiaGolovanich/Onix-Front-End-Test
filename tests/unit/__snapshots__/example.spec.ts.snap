// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld.vue renders props.msg when passed 1`] = `
<section
  class="news"
  msg="new message"
>
  <div
    class="current-page"
  >
    <img
      alt="page not found"
    />
    <h1
      class="not-found"
    >
      Not Found
    </h1>
    <p
      class="not-found"
    >
      Oops we couldn't find that page. Try going 
      <router-link
        to="[object Object]"
      >
        home
      </router-link>
    </p>
  </div>
</section>
`;
